### ISSUE TITLE
Map zoom size

### ISSUE THREAD

**Issue by xerion79585** [qvarforth/trackdirect#44]
The default web map zoom size can only be up to 16. Can it be larger?

**Comment by qvarforth** [comment]
Yes, it can be larger. Look in Map.js, you will easily find where to change maxZoom.
